wgengine/magicsock/magicsock_test.go

```diff
@@ -296,10 +297,19 @@ func meshStacks(logf logger.Logf, mutateNetmap func(idx int, nm *netmap.NetworkM
 				panic(fmt.Sprintf("failed to construct wgcfg from netmap: %v", err))
 			}
 			if err := m.Reconfig(wg); err != nil {
-				if ctx.Err() != nil || errors.Is(err, errConnClosed) {
+				if ctx.Err() != nil {
 					// shutdown race, don't care.
 					return
 				}
+				var me multierror.MultipleErrors
+				if errors.As(err, &me) {
+					for _, e := range me {
+						if errors.Is(e, errConnClosed) {
```

Comment:
woah, does an errors.Is not see into a multierror? that seems wrong. does unwrap make it impossible to implement an iterator?